Desktop/html/js/analyses.js

```diff
@@ -122,6 +130,23 @@ JASPWidgets.Analyses = JASPWidgets.View.extend({
 
 	reRender: function() {
 		this.analyses.forEach(function(analysis) {analysis.render();});
+		this.setBottomSpacerHeight();
+	},
+
+	setBottomSpacerHeight: function() {
+		var bottomSpacer = $("#bottomSpacer");
+		var height		 = window.innerHeight;
```

Comment:
Use better $(windows).innerHeight(): jQuery takes better into account the scrollbars.
I have noticed that when the horizontal scrollbar appears, then the bottomSpacer becomes a bit too high.